src/main/resources/static/api.yml

```diff
@@ -76,6 +76,40 @@ paths:
         500:
           $ref: '#/components/responses/500Response'
       x-codegen-request-body-name: body
+  /premises/summary:
+    get:
+      tags:
+        - Premises
+      summary: Returns a list of premises
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: false
```

Comment:
I think this should be `required: true` as both the frontend clients provide this header by default and it was made optional for backwards compatibility at the time.